components/playground.tsx

```diff
@@ -236,17 +263,191 @@ export default function Playground({
 		<main className="grid lg:grid-cols-[2fr_3fr] gap-20 lg:gap-12 w-full max-w-4xl">
 			<Form {...form}>
 				<form
+					id="passportform"
 					onSubmit={form.handleSubmit(onSubmit)}
 					className="w-full flex flex-col gap-y-6"
 				>
+					{userId ? (<p>1. Register</p>) : (<></>)}
+
+					{userId && guildMember !== undefined ? (
+						<span>
+							<FormField
+								control={form.control}
+								name="sendToDb"
+								render={({ field }) => (
+									<FormItem className="space-y-3">
+										<FormLabel>Register this passport</FormLabel>
+										<FormDescription>
+											Register for an hour-long passport ceremony to turn your data page into a real-life passport.
+										</FormDescription>
+										<FormControl>
+											<RadioGroup
+												onValueChange={(e) => {
+													field.onChange(e);
+													{
+														if (e == "false") {
+															setGenerationSteps(GENERATION_STEPS.base);
+														} else {
+															setGenerationSteps(GENERATION_STEPS.register);
+														}
+													}
+												}}
+												defaultValue={field.value}
+												className="flex flex-col space-y-1"
+											>
+												<FormItem className="flex space-x-1 space-y-0">
+													<span className="flex space-x-1 space-y-0">
+														<FormControl>
+															<RadioGroupItem value="true" />
+														</FormControl>
+														<FormLabel className="font-normal">Yes</FormLabel>
+													</span>
+												</FormItem>
+												<FormItem className="flex space-x-1 space-y-0">
+													<span className="flex space-x-1 space-y-0">
+														<FormControl>
+															<RadioGroupItem value="false" />
+														</FormControl>
+														<FormLabel className="font-normal">No</FormLabel>
```

Comment:
<img width="382" alt="Screenshot 2024-09-24 at 9 03 54 PM" src="https://github.com/user-attachments/assets/ed4f1f13-252a-45d7-907e-f24f5e511daf">

Nit, non-blocking: it just occurred to me that this is weird. It should be a single radio, or a button that you can toggle, or something, instead of a "yes" or "no" radio. There's no reason to check "no" except maybe to undo a "yes".

Reply:
Resolved in [3480db5](https://github.com/purduehackers/passport-issuing-office/pull/52/commits/3480db5711f75ef911b6a079e0d9deb8036ef6a8), if the switch is acceptable

Reply:
![image](https://github.com/user-attachments/assets/7d03706d-7f1f-4a91-be1c-1851581de895)
![image](https://github.com/user-attachments/assets/667a075c-e4a5-4ea7-aab8-0e259e812017)

Reply:
Switch is a great idea, but it's hard to see when it's in an off state